xml/decoders/soundtraxx/OEMBachmannCV.xml

```diff
@@ -0,0 +1,1524 @@
+        <label>F10 controls output Short</label>
+    </variable>
+    <variable item="F10 controls output 11" CV="44" mask="XXXXVXXX" default="0" minFn="10">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F10 controls output Dimmer</label>
+    </variable>
+    <variable item="F10 controls output 12" CV="44" mask="XXXVXXXX" default="0" minFn="10">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F10 controls output Mute</label>
+    </variable>
+    <variable item="F10 controls output 15" CV="44" mask="VXXXXXXX" default="0" minFn="10">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F10 controls output Coupler</label>
+    </variable>
+    <variable item="F11 controls output 8" CV="45" mask="XXXXXXXV" default="0" minFn="11">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F11 controls output Short</label>
+    </variable>
+    <variable item="F11 controls output 11" CV="45" mask="XXXXVXXX" default="0" minFn="11">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F11 controls output Dimmer</label>
+    </variable>
+    <variable item="F11 controls output 12" CV="45" mask="XXXVXXXX" default="0" minFn="11">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F11 controls output Mute</label>
+    </variable>
+    <variable item="F11 controls output 15" CV="45" mask="VXXXXXXX" default="0" minFn="11">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F11 controls output Coupler</label>
+    </variable>
+    <variable item="F12 controls output 8" CV="46" mask="XXXXXXXV" default="0" minFn="12">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F12 controls output Short</label>
+    </variable>
+    <variable item="F12 controls output 11" CV="46" mask="XXXXVXXX" default="0" minFn="12">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F12 controls output Dimmer</label>
+    </variable>
+    <variable item="F12 controls output 12" CV="46" mask="XXXVXXXX" default="0" minFn="12">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F12 controls output Mute</label>
+    </variable>
+    <variable item="F12 controls output 15" CV="46" mask="VXXXXXXX" default="0" minFn="12">
+        <xi:include href="http://jmri.org/xml/decoders/parts/enum-NoYes.xml"/>
+        <label>F12 controls output Coupler</label>
+    </variable>
+    <!-- Lighting Configuration follows -->
+    <variable item="Headlight F0(f) Effect Selection" CV="49" mask="XXXVVVVV" default="0" tooltip="Determines the effect generated from the headlight">
+      <defaultItem default="17" include="1028"/>
```

Comment:
The schema error is this line.  I think, but I haven't confirmed, that updating the schema reference will fix the error.

Reply:
I'm sure it will fix this one, and probably others.

The `defaultItem` element wasn't defined in decoder.xsd. It is defined in `decoder-4-15-2.xsd`.